### DISCUSSION TITLE
Blog unqueues after a short time of doing nothing

### DISCUSSION THREAD

**Discussion by DaymareChimera** [TumblThreeApp/TumblThree#495] [General]
**Describe the bug**
I'd been spending a few days downloading a Twitter blog (authenticated), and all was going well up until yesterday. I tried to queue up the blog again and resume the download process, but after about 30 seconds it evaluated 20 post URLS and then automatically un-queued itself. A few times, I managed to get it to successfully download a few posts, but now it does the above behavior and downloads nothing new. Checking the blog's online status hasn't resulted in any blog offline errors while this has been happening. This continues happening after updating to the most recent version of the application.

1. Enter a Twitter URL and click Add Blog.
2. Select the blog from the list and right-click. 
3. Click "Enqueue selection."
4. Click the Download button.
5. Go to the queue tab.
6. See error

**Expected behavior**
I expected it to continue evaluating post urls and downloading new posts until the connection timed out or it finished. Previously, the blog had stayed in the queue even when encountering connection timeouts, and would wait until it was able to start downloading again.

**Screenshots**
![image](https://github.com/TumblThreeApp/TumblThree/assets/149974122/6c1a46e3-ce9b-4944-8c82-4347d496965e)
![image](https://github.com/TumblThreeApp/TumblThree/assets/149974122/95de7e44-bed7-4357-9fcc-3059378edebd)

**Desktop (please complete the following information):**
 - TumblThree version: v2.11.4
 - OS: Windows 10 Home
 - Browser: Opera
 - Version: 104.0.4944.36 (Opera), 2H22 (Windows)

**Additional context**
OS running in VirtualBox. Still a problem after restarting the OS, restarting VirtualBox, and restarting my computer.

**Comment by thomas694** [comment]
Please look up the value after the field "LastId" in the .twitter file inside the Index folder.

**Comment by DaymareChimera** [comment]
It's this: 1721225519453077824

**Comment by thomas694** [comment]
I already assumed. The whole blog was already scanned once. From now on it only scans new posts since the last successful scan. After a few hours or a day, it may download a couple of (new) posts. If you add it to the queue again before the blog has new posts, it just finishes after a short while.
You should be able to check that in your explorer folder by sorting on modified date.

**Comment by DaymareChimera** [comment]
Oh okay, thank you! I'll check that out and report back!

**Comment by DaymareChimera** [comment]
Okay, update: I tried with a different account, but ran into the same issue where I only got ~6k posts off an account that Twitter shows as having ~81k posts. Interestingly, this only seemed to start after I checked "Download replies" in the Details section after it started running. I deleted the files downloaded, tried again with checking the box before I queued it up, and it worked fine.

**Comment by DaymareChimera** [comment]
Update: It appears to sometimes stop randomly and demonstrate the above behavior, with the progress bar in details displaying as not full. (I don't know if that's only a visual glitch, or indicative of a true error in the scanning process?)
Checking "force rescan" and re-queueing sort of works in the meantime.

**Comment by DaymareChimera** [comment]
Hmm. Does "number of downloadable items" indicate what has yet to be downloaded, or what can be downloaded overall, regardless of if it has or hasn't yet?

**Reply by thomas694** [reply]
It indicates what can be downloaded overall (depending on the selected options).

**Comment by DaymareChimera** [comment]
And now I'm running into this? Where it quits downloading just before it reaches all downloadable items. Forcing a rescan doesn't seem to fix the issue, nor does queueing and unqueueing. The progress bar displays as almost full, except for a tiny sliver at the end. Putting this here, because I wonder if it's related.

(And some of them are way over, which I think is because I rescanned? Unsure.)
![image](https://github.com/TumblThreeApp/TumblThree/assets/149974122/3b6e4a9e-809c-4c8f-8c27-cfd1c7308654)
